making-decisions/README.md

```diff
@@ -0,0 +1,85 @@
+Although that's not what we need here, we need to be able to make quick
+democratic decisions within one sprint.
+
+I am amazed that the framework champions these type of questions and
+suggestions which can help us decide:
+
+- Whose problem are we trying to solve?
+- Who else could be impacted?
+- Who has solved a similar problem?
+- Who is likely to disagree, dissent, reject, or opt out? Who else may care?
+- Gather input from internal customers and those who you will need help from (surveys, interviews, focus groups, etc.)
+- Remain open to new information and perspectives
+- Consider peer-to-peer feedback and communication options in addition to formal channels
+- How will we monitor mailing lists and other feedback channels after the launch?
+- How willing are we to make revisions based on feedback?
+- What's a reasonable window of time for additional input and refinement?
+- Did we overlook something important? How do we address it?
+
+## [FESCo](https://docs.fedoraproject.org/en-US/fesco/)
+
+Decisions in FESCo are the daily bread so we can get an inspiration here. They
+have a well-thought voting mechanism:
+
+- One week for FESCo members to vote `+1` or `-1`. After the period accept if 3
+  `+1`s were received without a negative vote and vice versa.
+
+- Immediately accept or reject when getting 7 votes of the same type.
+
+- After a week, discuss the topic and decide.
+
+This also sounds like a too complicated solution for us although I like the
+aspect of: discuss when unsure, accept/reject in agreement.
+
+## [UK Government suggestion for voting in charities](https://www.gov.uk/guidance/charity-meetings-making-decisions-and-voting#follow-voting-rules-if-applicable)
+
+Simple and clear: members cast votes on meetings :) just like we do with grooming.
+
+## ["Board Voting: Common Steps & Tips for Better Decision-Making"](https://boardable.com/blog/board-voting/)
+
+Interesting (and biased) blog post which describes in detail the process of
+voting in non-profit organizations.
+
+## [Franta's suggestion about decision making in Teal organizations](https://reinventingorganizationswiki.com/theory/decision-making/)
+
+A thorough and well-explained comparison between types of organization and the
+decision making process followed by a fascinating read about the Teal decision
+making.
+
+Funnily enough, the advice the article is giving is contrary to my current
+proposal ('duh). They advise for the decision maker to make a call based on
+research, advices and being the subject matter expert. I instead suggested to
+have a consensus on a decision. The argument from the article is that reaching
+consensus is too expensive time-wise and resource-wise. I'd still try what I
+propose and reevaluate what's the most optimal process for us after a period of
+time.
+
+We can definitely still learn from the article so I strongly suggest reading it.
+
+## [Consent Decision Making](https://thedecider.app/consent-decision-making)
```

Comment:
I like this the most, and something similar is also mentioned in the Teal organization description above.

It gives room for discussions (which the current proposal doesn't mention how they should happen) and makes objections a part of the process, while also making the whole thing a little bit faster by not requiring full consensus.